app/blueprints/api/v3/category_blueprint.rb

```diff
@@ -1,5 +1,5 @@
 class Api::V3::CategoryBlueprint < Blueprinter::Base
   fields :id, :name, :created_at, :updated_at
 
-  association :game, blueprint: Api::V3::GameBlueprint, if: ->(_, options) { options[:toplevel] == :category }
+  association :game, blueprint: Api::V3::GameBlueprint, if: ->(_, _, options) { options[:toplevel] == :category }
```

Comment:
Blueprinter has a depreciation warning now, because they changed the conditionals to take 3 parameters now instead of 2.